DECISION-MAKING.md

```diff
@@ -6,6 +6,13 @@ This document list which team or people are **responsible for making decisions**
 
 All decisions are taken by consulting various parties of different teams in and outside of Mozilla.
 
+Owners are responsible for
+
+- Providing road maps for their module that align with Mozilla OKRs
```

Comment:
are Mozilla OKRs the super top level OKRs or the one the product owners set for IAM and other products?
if thats the former, then I don't think we're able to have per-module roadmap (these would end up disjointed), if thats the later then its effectively a matter of *setting milestones with dates* based on the OKR (and as per other line items below also decompose the OKRs are necessary)

I would be in favor of this, in particular, if we word it as "milestones with dates" (since everyone is always understanding roadmaps in a different manner)

Reply:
I was thinking of formalizing the practice that you've started of setting IAM OKRs.

Reply:
i met with all service/modules owners in infosec and set OKRs with them basically, so that i could derive milestones and some kind of roadmap (instead of having each module owner do their own roadmap)

Reply:
makes sense to me!